File: main/src/main/java/net/fwitz/math/main/binary/escape/newton/NewtonsMethodZCubedMinus2ZMinus5.java
package net.fwitz.math.main.binary.escape.newton;

public class NewtonsMethodZCubedMinus2ZMinus5 {
    public static void main(String[] args) {
        NewtonsMethodPlot.render(
                "z^3 -2z - 5  (Newton's Method)",
                z -> z.pow3().minus(z.times(2)).minus(5),
                z -> z.pow2().times(3).minus(2));
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/escape/newton/NewtonsMethodZCubedMinus2ZMinus5.kt
package net.fwitz.math.main.binary.escape.newton

object NewtonsMethodZCubedMinus2ZMinus5 {
    @JvmStatic
    fun main(args: Array<String>) = NewtonsMethodPlot.render(
        name = "z^3 -2z - 5  (Newton's Method)",
        f = { z -> z.pow3 - z * 2 - 5 },
        df = { z -> z.pow2 * 3 - 2 }
    )
}